service/compute/commands.go

```diff
@@ -9,6 +9,63 @@ import (
 	"github.com/databricks/databricks-sdk-go/useragent"
 )
 
+type CommandExecutorV2 struct {
+	clustersAPI  *ClustersAPI
+	executionAPI *CommandExecutionAPI
+	language     Language
+	clusterID    string
+	contextID    string
+}
+
+// Start the command execution context on a cluster and ensure it transitions to a running state
+func (a *CommandExecutionAPI) Start(ctx context.Context, clusterID string, language Language) (*CommandExecutorV2, error) {
+	executionImpl, ok := a.impl.(*commandExecutionImpl)
+	if !ok {
+		return nil, fmt.Errorf("unknown command executionimplementation: %v", a.impl)
+	}
+	// re-initializing clusters API here, so that we have less IF's in the generator
+	clustersAPI := NewClusters(executionImpl.client)
+	err := clustersAPI.EnsureClusterIsRunning(ctx, clusterID)
+	if err != nil {
+		return nil, err
+	}
+	commandContext, err := a.CreateAndWait(ctx, CreateContext{
+		ClusterId: clusterID,
+		Language:  language,
+	})
+	if err != nil {
+		return nil, err
+	}
+	return &CommandExecutorV2{
+		clustersAPI:  clustersAPI,
+		executionAPI: a,
+		language:     language,
+		clusterID:    clusterID,
+		contextID:    commandContext.Id,
+	}, nil
+}
+
+func (c *CommandExecutorV2) Destroy(ctx context.Context) error {
+	return c.executionAPI.Destroy(ctx, DestroyContext{
+		ClusterId: c.clusterID,
+		ContextId: c.contextID,
+	})
+}
+
+// Execute runs given command in the running cluster and context and waits for the results
+func (c *CommandExecutorV2) Execute(ctx context.Context, command string) (*Results, error) {
+	status, err := c.executionAPI.ExecuteAndWait(ctx, Command{
```

Comment:
These calls should set the `sdk-feature` like the previous ones.

Reply:
`WaitCommandStatusCommandExecutionFinishedOrError` adds it already